docs/abi.md

```diff
@@ -179,6 +179,11 @@ If writing to the specified channel would violate
 - `param[4]: u32`: Source handle array count
 - `result[0]: u32`: Status of operation
 
+### `channel_write_with_privilege`
+
+The same as [`channel_write`], except that it uses the current node's
```

Comment:
I don't think this works as a link, does it?

Reply:
Updated to include anchor link.